### PR TITLE
Configure the content-header of template

### PR DESCRIPTION
**Pull request by warquia**
| Question                | Answer
| ----------------------- | [-----------------------]
| Issue or Enhancement    | Enhancement
| License                 | MIT

#### What's in this PR?

Allowing you to generate or not a content-header, thus allowing at 1024x768 resolutions gains more screen space,

#### Checklist

- [x] I tested these changes.
Git

### CODE REVIEW

**Review by resslinger — CHANGES_REQUESTED** [review]
Please add a fallback, if the config var is not set.

**Review by warquia — COMMENTED** [review]
add fallback

**Comment by warquia** [comment]
a

**Comment by Shidersz** [comment]
@warquia what classes did you have added to the new config option you have created to get from **picture 1** to **picture 2**. If the goal is to not shown the `content-header` when the section is not used, we can do like next:

```blade
{{-- Content Header --}}
@hasSection('content-header')
    <div class="content-header">
        <div class="{{ config('adminlte.classes_content_header') ?: $def_container_class }}">
            @yield('content_header')
        </div>
    </div>
@endif
```

Also, note when you do:

```blade
<div class="{{ config('adminlte.classes_content_header_main' , 'content-header') }}">
```

the default value will only be used when the configuration option do not exists. So, when defining the option to `null` it will inject a null on the class and may result on undesired behavior. You can read [laravel docs](https://laravel.com/docs/8.x/helpers#method-config) for details. For this case, I believe it will be better to use:

```blade
<div class="{{ config('adminlte.classes_content_header_main') ?: 'content-header' }}">
```

**Comment by warquia** [comment]
Good morning everyone, really has-section is the best approach, I will correct the pull-request with your analysis.
By the way, I'm new to git so I'm going to send this pull in master yet but next time I'll follow the guidance using a branch

Thank you all

**Review by Shidersz — CHANGES_REQUESTED** [review]
Please, do not delete `config/adminlte.php` file. Just use `git checkout config/adminlte.php` to restore it to the original one.

File: resources/views/page.blade.php [hunk 1]
```diff
@@ -36,11 +36,13 @@
         <div class="content-wrapper {{ config('adminlte.classes_content_wrapper') ?? '' }}">
 
             {{-- Content Header --}}
+            @hasSection('content_header')
```
Comment (Shidersz) [inline resources/views/page.blade.php:39]: Add an indent of 4 spaces to the inner code here.

**Review by Shidersz — APPROVED** [review]

**Comment by Shidersz** [comment]
@resslinger @warquia This PR is ok for me now...

**Review by resslinger — APPROVED** [review]

**Pull request merged**